applications/StructuralMechanicsApplication/custom_constitutive/linear_isotropic_damage_3D_law.h

```diff
@@ -0,0 +1,334 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Marcelo Raschi
+//  Collaborator:
+
+#if !defined(KRATOS_LINEAR_ISOTROPIC_DAMAGE_3D_LAW_H_INCLUDED)
+#define KRATOS_LINEAR_ISOTROPIC_DAMAGE_3D_LAW_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/checks.h"
```

Comment:
This should be imported in the cpp, from what I can see it is not used in the header file

Reply:
done